scripts/template.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
```

Comment:
Nitpick/non-blocking suggestion: add `xmlns` so that thymeleaf attributes (`th:`) are recognized by IntelliJ. It's not required, just a nice to have for devs.

```suggestion
<html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
```